src/webgl/light.js

```diff
@@ -832,4 +832,64 @@ p5.prototype.spotLight = function(
   return this;
 };
 
+/**
+ * This function will remove all the lights from the sketch for the
+ * subsequent materials rendered. It affects all the subsequent methods.
```

Comment:
Can we put a note in here about how to re-enable lights?

Reply:
Something like this: `To Re-enable lights, you need to call the lighting methods again`

Reply:
How about "Calls to lighting methods made after `noLights()` will re-enable lights in the scene."